frontend/src/app/modules/global_search/global-search-input.component.ts

```diff
@@ -125,10 +126,29 @@ export class GlobalSearchInputComponent implements OnInit, OnDestroy {
 
     // handle click on search button
     if (ContainHelpers.insideOrSelf(this.btn.nativeElement, event.target)) {
-      this.submitNonEmptySearch();
+      if (this.isMobile) {
+        this.toggleMobileSearch();
+        // open ng-select menu on default
+        jQuery('.ng-input input').focus();
+      } else {
+        this.submitNonEmptySearch();
+      }
+    }
+  }
+
+  @HostListener('window:resize', ['$event'])
+  onResize(event:any) {
+    if (event.target.innerWidth > 680 && this.mobileSearch) {
```

Comment:
Here, as commented below for the `isMobile` getter, it might be a good idea to extract the threshold of `680` to a service, and use it as `Device.mobileWidthTheshold`